presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -231,6 +232,8 @@
     public static final String PREFER_MERGE_JOIN = "prefer_merge_join";
     public static final String SEGMENTED_AGGREGATION_ENABLED = "segmented_aggregation_enabled";
     public static final String USE_EXTERNAL_PLAN_STATISTICS = "use_external_plan_statistics";
+    public static final String NODE_NUM_IN_LOGICAL_PLAN_EXCEED_THD_EXCEPTION_ENABLED = "node_num_in_logical_plan_exceed_thd_exception_enabled";
+    public static final String MAX_NODE_NUM_IN_LOGICAL_PLAN = "max_node_num_in_logical_plan";
```

Comment:
`logical_plan_leaves_limit` or `max_leaves_logical_plan` is more suitable, since we're tracking just leaves here

Reply:
Agree, if we agree that counting number of leaves is enough here.